app-text/multitail/multitail-7.0.0.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake flag-o-matic optfeature toolchain-funcs
+
+DESCRIPTION="Tail with multiple windows"
+HOMEPAGE="http://www.vanheusden.com/multitail/ https://github.com/folkertvanheusden/multitail/"
+SRC_URI="https://github.com/folkertvanheusden/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux"
+IUSE="debug examples unicode"
+RESTRICT="test" # bug 492270
+
+RDEPEND="sys-libs/ncurses:=[unicode(+)?]"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=(
+  "${FILESDIR}"/${PN}-7.0.0-gentoo.patch
+  "${FILESDIR}"/${PN}-7.0.0-fix-clang16-build.patch
+  "${FILESDIR}"/${PN}-7.0.0-fix-lto-type-mismatch.patch
+)
+
+src_prepare() {
+	cmake_src_prepare
+	# cmake looks for licence.txt to install it, which does not exist in the package
+	cp LICENSE license.txt
+	sed \
+		-e "/^DESTDIR/s:=.*$:=${EPREFIX}:g" \
+		-i Makefile || die
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DUTF8_SUPPORT=$(usex unicode)
+	)
+	CMAKE_BUILD_TYPE=$(usex debug Debug)
+
+	cmake_src_configure
+}
+
+src_install() {
```

Comment:
Do you need to delete the LICENSE file in src_install? We don't want to install it as it's covered by LICENSE (the variable) in the ebuild.

Reply:
Yup, this is what `rm -rf "${ED}"/usr/share/doc/multitail-VERSION=6.4.3 || die` does. We actually delete all the stuff this does with docs and put do our own. 
Licenes.txt is installed into this directory:
```
-- Installing: /var/tmp/portage/app-text/multitail-7.0.0/image/usr/share/doc/multitail-VERSION=6.4.3/license.txt
```